tests/phpunit/CRM/Financial/BAO/ExportFormat/DataProvider/QuickbooksCSVProviderTest.php

```diff
@@ -51,8 +51,10 @@ public function testMemoColumnEqualsInvoiceNumber() {
   public function testNameColumnEqualsContactId() {
     $data = $this->createTestContributionAndGenerateBatchRows();
 
-    $this->assertEquals($data['batch_rows'][0]['Name'], $data['contribution_data']['contact_id']);
-    $this->assertEquals($data['batch_rows'][1]['Name'], $data['contribution_data']['contact_id']);
+    $exportName = $data['contribution_data']['contact_id'] . ' - ' . $data['contribution_data']['contact_name'];
```

Comment:
given this change, the test method name no longer represents what it tests, so it should be updated to reflect that, so you can maybe change it form:

`testNameColumnEqualsContactId`

to

`testNameColumnEqualsContactDisplayNamePlusContactId`

Reply:
Update test function name.